packages/components/src/components/breadcrumb/breadcrumb-item.tsx

```diff
@@ -0,0 +1,79 @@
+import { Component, h, Element, State, Listen } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-breadcrumb-item',
+  styleUrl: 'breadcrumb-item.scss',
+  shadow: true,
+})
+export class BreadcrumbItem {
+
+  @State() isLastItem: boolean = false;
+  @Element() el;
+
+  @Listen('mousedown', { target: 'document' })
```

Comment:
Usage of 'mousedown' event listener: You are listening for a 'mousedown' event at the document level to detect if the click happened outside the component. The problem is, 'mousedown' will be triggered even when users are selecting text or clicking and holding mouse button down, which might not be the desired behavior. A 'click' event might be more suitable in this case.